resources/js/Shared/Layout/Nav.vue

```diff
@@ -128,97 +129,130 @@ defineExpose({
         <div v-if="isLoginFormSelected" class="rounded-lg px-6 pb-8">
           <form class="space-y-5" @submit.prevent="login">
             <div>
-              <label class="mb-1 block text-sm font-semibold text-gray-800">Email</label>
-              <input v-model="loginForm.email" type="email" class="w-full rounded-lg border-blumilk-200 py-3 md:p-2" required>
+              <label class="mb-1 block text-sm font-semibold text-gray-800">{{ __('Email') }}</label>
+              <input v-model="loginForm.email" type="email" class="w-full rounded-lg border-blumilk-200 py-3 md:p-2"
+                     required
+              >
             </div>
             <div>
-              <label class="mb-1 block text-sm font-semibold text-gray-800">Password</label>
-              <input v-model="loginForm.password" type="password" class="w-full rounded-lg border-blumilk-200 py-3 md:p-2" required>
+              <label class="mb-1 block text-sm font-semibold text-gray-800">{{ __('Password') }}</label>
+              <input v-model="loginForm.password" type="password" class="w-full rounded-lg border-blumilk-200 py-3 md:p-2"
+                     required
+              >
               <ErrorMessage :message="loginForm.errors.loginError" />
             </div>
             <div class="flex w-full md:w-fit">
-              <button type="submit" class="w-full rounded-lg bg-blumilk-500 p-4 font-semibold text-white hover:bg-blumilk-600 md:py-2">
-                Log in
+              <button type="submit"
+                      class="w-full rounded-lg bg-blumilk-500 p-4 font-semibold text-white hover:bg-blumilk-600 md:py-2"
+              >
+                {{ __('Log in') }}
               </button>
             </div>
           </form>
           <button :disabled="loginForm.processing" class="mt-6 text-xs font-light" @click="toggleAuthOption">
-            Don't have an account? <span class="font-normal">Sign up</span>
+            {{ __('No account') }} <span class="font-normal">{{ __('Sign up') }}</span>
```

Comment:
I think it should stay as it was before. Remember about translation.

Reply:
fixed